modules/installation-creating-gcp-shared-vpc-cluster-wide-firewall-rules.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-shared-vpc-cluster-wide-firewall-rules_{context}"]
+= Creating cluster-wide firewall rules for a shared VPC in GCP
+
+You can create cluster-wide firewall rules to allow the access that the {product-title} cluster requires.
+
+.Prerequisites
+
+* You exported the variables that the Deployment Manager templates require to deploy your cluster.
+* You created the networking and load balancing components in GCP that your cluster requires.
+
+.Procedure
+
+. Add a single firewall rule to allow the Google Cloud Engine health checks to access all of the services. This rule enables the ingress load balancers to determine the health status of their instances.
++
+----
+$ gcloud compute firewall-rules create --allow='tcp:30000-32767,udp:30000-32767' --network="${CLUSTER_NETWORK}" --source-ranges='130.211.0.0/22,35.191.0.0/16,209.85.152.0/22,209.85.204.0/22' --target-tags="${INFRA_ID}-master,${INFRA_ID}-worker" ${INFRA_ID}-ingress-hc
+----
+
+. Add a single firewall rule to allow access to all cluster services:
++
+----
+$ gcloud compute firewall-rules create --allow='tcp:80,tcp:443' --network="${CLUSTER_NETWORK}" --source-ranges="0.0.0.0/0" --target-tags="${INFRA_ID}-master,${INFRA_ID}-worker" ${INFRA_ID}-ingress
```

Comment:
Again, please add --account=${HOST_PROJECT_ACCOUNT} --project=${HOST_PROJECT} to this command line.